monkey/monkey_island/cc/setup/data_dir.py

```diff
@@ -90,3 +92,23 @@ def _data_dir_version_mismatch_exists(data_dir_path: Path) -> bool:
     island_version = get_version()
 
     return island_version != data_dir_version
+
+
+def _copy_plugins_into_data_dir(data_dir_path: Path):
+    plugin_path = Path(MONKEY_ISLAND_ABS_PATH) / "plugins"
+    try:
+        plugins_destination_path = _create_plugins_dir(data_dir_path)
+
+        plugin_tar_paths = list(plugin_path.glob("*.tar"))
+        for plugin_tar_path in plugin_tar_paths:
+            shutil.copy2(plugin_tar_path, plugins_destination_path)
```

Comment:
Why `copy2()`?

Reply:
<img width="880" alt="image" src="https://user-images.githubusercontent.com/15820737/215754187-f377b59d-570a-4e1c-8033-fcbe967c3b77.png">

Reply:
Why do we need to preserve metadata?